### PR TITLE
add sort_weight to order_frequency table

### PR DESCRIPTION
**Pull request by 22Sandu**

### CODE REVIEW

File: api/src/main/java/org/openmrs/OrderFrequency.java [hunk 1]
```diff
@@ -115,6 +117,17 @@ public String getDescription() {
 		}
 		return null;
 	}
+        
+        public Double getSortWeight() {
+		return sortWeight;
+	}
+	
+	public void setSortWeight(Double sortWeight) {
+           // if(sortWeight != null)
```
Comment (dkayiwa) [inline api/src/main/java/org/openmrs/OrderFrequency.java]: Do you still need these commented out lines?

**Comment by dkayiwa** [comment]
@22Sandu it is great to always include the ticket number in the commit message. Hopefully you got  a chance to look at https://wiki.openmrs.org/display/docs/Pull+Request+Tips

**Comment by dkayiwa** [comment]
@22Sandu i have also just modified my second last comment on the JIRA ticket regarding unit testing.

File: api/src/main/resources/org/openmrs/api/db/hibernate/OrderFrequency.hbm.xml [hunk 2]
```diff
@@ -14,38 +14,34 @@
     "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
     "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
 <hibernate-mapping>
-	<class name="org.openmrs.OrderFrequency" table="order_frequency">
```
Comment (wluyima) [inline api/src/main/resources/org/openmrs/api/db/hibernate/OrderFrequency.hbm.xml]: Seems like this file has been formatted? Can you try undoing the changes in it? Then just add the single for the new field and see if that resolves it

File: api/src/main/resources/org/openmrs/api/db/hibernate/OrderFrequency.hbm.xml [hunk 3]
```diff
@@ -14,38 +14,34 @@
...
+        <id name="orderFrequencyId" type="java.lang.Integer" column="order_frequency_id">
+            <generator class="native">
+                <param name="sequence">order_frequency_id_seq</param>
+            </generator>
+        </id>
+        <many-to-one name="concept" class="org.openmrs.Concept"
+                             not-null="true" column="concept_id" unique="true" />
+        <property name="frequencyPerDay" type="double" column="frequency_per_day"
+                          length="22" />
+        <property name="sortWeight" type="double" column="sort_weight" length="22" />
```
Comment (wluyima) [inline api/src/main/resources/org/openmrs/api/db/hibernate/OrderFrequency.hbm.xml]: My assumption is that hibernate can resolve the field length for primitive types and their wrapper classes

**Comment by dkayiwa** [comment]
@22Sandu are you still working on this?

**Comment by wluyima** [comment]
Did you see my comment on the ticket? I don't think we need to add this column, I think they can implement this with a global property

**Comment by dkayiwa** [comment]
Closing this because the reporter decided to do it differently as per the ticket comments.

**Pull request closed without merging**